package.json

```diff
@@ -26,7 +26,7 @@
     "Bernie Zang <nzang@cisco.com>",
     "Vagou Zhou <vagouz@cisco.com>"
   ],
-  "main": "src/index.js",
+  "main": "packages/recipe-private-web-client/dist/index.js",
```

Comment:
Please don't do this. The web client should reference `@ciscospark/recipe-private-web-client`, not `spark-js-sdk`